client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -212,13 +217,15 @@ export class PlanUpgradeUpsell extends PureComponent {
 						<p>
 							<b>
 								{ translate(
-									'Upgrade to the Premium plan and access nearly 200 premium themes for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more.',
+									'Upgrade to the Premium plan and access over 250 premium themes for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.